.github/gnav.md

```diff
@@ -0,0 +1,26 @@
+## Description
```

Comment:
Why do we need this file?

Reply:
We don't, it looks like the branch needs a sync - https://github.com/adobecom/milo/pull/1233